### PR TITLE
feat: create cron to cleanup unverified work emails

### PR DESCRIPTION
**Pull request by omBratteng**
Auto-cleanups un-verified work emails that are older than one hour.

AS-538

### CODE REVIEW

**Review by rebelchris — APPROVED** [review]
Looks good to me :)

File: __tests__/cron/cleanZombieUserCompany.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    {
+      userId: '1',
+      verified: true,
+      email: 'u1@com1.com',
+      code: '123',
+    },
+  ]);
+});
+
+describe('cleanZombieUserCompany', () => {
```
Comment (capJavert) [inline __tests__/cron/cleanZombieUserCompany.ts:29]: I see you registered it but add `should be registered` test as well (example in daily digest cron)

**Review by capJavert — APPROVED** [review]
🤘

File: __tests__/cron/cleanZombieUserCompany.ts (hunk 1 above)
Comment (omBratteng) [reply __tests__/cron/cleanZombieUserCompany.ts:30]: Added registered test too

**Pull request merged**